Notebooks/Bruxism_detection.ipynb

```diff
@@ -2,24 +2,23 @@
  "cells": [
```

Comment:
<p><code>Impedance_thresholding</code> doesn't use <code>picks_chan</code> parameter. Please remove that in your function and in this code. Also, I advice to rename to </p><p><code>Impedance_thresholding_sliding</code> (and refactoring corresponding <code>test_*.py</code>) to emphasize the fact that it does an epoching.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lkorczowski/Tinnitus-n-Sleep/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Ok!